.github/workflows/multi_service_parallel_build_gha_lint.yml

```diff
@@ -11,7 +11,7 @@ on:
 env:
   TAG: ${{ github.ref }}
   REPO_NAME: ${{ github.repository }}
-  ARGO_NAMESPACE: wl-vt-cnask-dev
+  ARGO_NAMESPACE: wl-<WL_NAME>-buil
```

Comment:
mistyping here, think it must be wl-<WL_NAME>-buil**d**

Reply:
fixed here https://github.com/CloudGeometry/cg-devx-wl-cnask/pull/2